File: app/src/main/java/io/nanodegree/andrea/popularmovies/model/Review.java
package io.nanodegree.andrea.popularmovies.model;

import com.squareup.moshi.Json;

/**
 * Created by Andrea Loddo (andrea@evenly.io) on 20.12.2018
 * for Evenly GmbH
 * <p>
 * Copyright (c) 2018 Evenly GmbH,
 * all rights reserved
 */
public class Review {

    @Json(name = "id")
    public String id;

    @Json(name = "author")
    public String author;

    @Json(name = "content")
    public String content;

    @Json(name = "url")
    public String url;
}

File: feature_movie/src/main/java/java/io/nanodegree/popularmovies/feature/movie/data/model/Review.kt
package java.io.nanodegree.popularmovies.feature.movie.data.model

import com.google.gson.annotations.SerializedName

/**
 * Created by Andrea Loddo (andrea@evenly.io) on 20.12.2018
 * for Evenly GmbH
 *
 *
 * Copyright (c) 2018 Evenly GmbH,
 * all rights reserved
 */
data class Review(@SerializedName("id")
                  val id: String? = null,

                  @SerializedName("author")
                  val author: String? = null,

                  @SerializedName("content")
                  val content: String? = null,

                  @SerializedName("url")
                  val url: String? = null)